src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
My thinking was that `usa-input-inline-error` would be added to the input with `usa-input-inline` so it would already have the width style.